hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/state/AuthnFailedErrorState.java

```diff
@@ -38,6 +37,8 @@ public AuthnFailedErrorState(
         return Optional.fromNullable(forceAuthentication);
     }
 
+    // Keep this for now to make deserialization with the previous version of FraudEventDetectedState compatible
+    // TODO: After this version has been released, remove the relayState property from this class
```

Comment:
Why do we want to remove it?

Reply:
Because the parent class has this property. The subclass is duplicating it